v1/charm-dev.yaml

```diff
@@ -60,6 +60,21 @@ instances:
           - snap install yq
           - snap refresh
 
+        write_files:
+        - path: "/etc/update-motd.d/99-use-beta-channels"
+          permissions: "0755"
+          content: |
+            #!/bin/sh
+
+            echo "======================================================================"
+            echo " If you're feeling adventurous, you could help the charming community"
+            echo " by switching to the beta channels and reporting any issues you "
+            echo " encounter in you journey!"
+            echo
+            echo "   sudo snap refresh juju --channel=3/beta"
+            echo "   sudo snap refresh charmcraft --classic --channel=3.x/beta"
```

Comment:
It looks like this channel is currently closed. Can you confirm if this really is the intention?

```
$ sudo snap refresh charmcraft --classic --channel=3.x/beta
charmcraft (3.x/candidate) 3.1.1 from Canonical✓ refreshed
Channel 3.x/beta for charmcraft is closed; temporarily forwarding to 3.x/candidate.
```

Reply:
Huh. I think it worked yesterday (?), not sure.
Yes, I'll change to candidate.

Reply:
Ok odd, `snap info` looks fine. Let's keep at beta?
```
  3.x/stable:       –                                           
  3.x/candidate:    3.1.1                2024-07-24 (4264) 62MB classic
  3.x/beta:         ↑                                           
  3.x/edge:         3.1.1.post4+g1330582 2024-07-31 (4318) 62MB classic
```

Reply:
Right, that's what it looks like when the channel is closed, it just forwards to the next "more stable" channel. And that worked fine. If something is later published to beta, the snap will update to that. 

Anyway, I am fine with beta. I mainly wanted to be sure that you didn't mean to have candidate or edge instead.